src/react-contenteditable.js

```diff
@@ -7,20 +7,18 @@ export default class ContentEditable extends React.Component {
   }
 
   render() {
-    var props = {};
-    Object.assign(props, this.props);
-    delete props.tagName;
-    delete props.html;
```

Comment:
@WendellLiu he means that the contentEditable element is passed props tagName and html while it should not.

But @lovasoa I think it's a mistake. Doing `var { tagName, html, onChange, ...props } = this.props;` means that `props` won't contain tagName, html or onChange, only the "rest" so there's no need to do the delete anymore